hack/internal/pkg/installconfig/base-install-config-amd64.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
```

Comment:
issue: By not importing `cluster-lifecycle` into our repository, we can effectively drop this whole `installconfig` package as well.